.github/workflows/clang-tidy-modern.sh

```diff
@@ -0,0 +1,2 @@
+#!/bin/bash
+run-clang-tidy-10.py -config='' -header-filter='.*' -fix
```

Comment:
What's your reason to move this to its own file? As long as it is just one line, I would leave it in tidy-modern.yml

Reply:
Done